src/unity/schemas/Account.yml

```diff
@@ -1,20 +1,9 @@
 properties:
   id:
-    type: string
-    description: account id
-  quartzId:
```

Comment:
when a resource comes from quartz, the id is the quartz id and the other id is the idpe id